src/edu/stanford/dlss/WasapiDownloader.java

```diff
@@ -0,0 +1,8 @@
+package edu.stanford.dlss;
```

Comment:
Can we create a web archiving package? E.g. src.edu.stanford.dlss.webarchiving or something like that? Like we did with edu.stanford.dlss.wowza for embed.